core/node/api_server/src/web3/tests/mod.rs

```diff
@@ -1102,3 +1109,122 @@ impl HttpTest for GenesisConfigTest {
 async fn tracing_genesis_config() {
     test_http_server(GenesisConfigTest).await;
 }
+
+#[derive(Debug)]
+struct GetBytecodeTest;
+
+impl GetBytecodeTest {
+    async fn insert_evm_bytecode(
+        connection: &mut Connection<'_, Core>,
+        at_block: L2BlockNumber,
+        address: Address,
+    ) -> anyhow::Result<()> {
+        let evm_bytecode_hash = hash_evm_bytecode(RAW_EVM_BYTECODE);
```

Comment:
@0xVolosnikov This doesn't matter in the context for this test, but in general: Does the padded / prefixed bytecode get hashed (seems like that from the EVM deploy tracer implementation), or the original one?

Reply:
Yes, padded and prefixed code. So we can enforce check that raw bytecode has odd amount of 32-byte words.